contracts/payloads/IGP106/PayloadIGP106.sol

```diff
@@ -0,0 +1,566 @@
+            IFluidVaultT1(wstUSR_USDC_VAULT).updateLiquidationThreshold(LT);
+        }
+
+        {
+            address wstUSR_USDT_VAULT = getVaultAddress(111);
+            IFluidVaultT1(wstUSR_USDT_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(wstUSR_USDT_VAULT).updateLiquidationThreshold(LT);
+        }
+
+        {
+            address wstUSR_GHO_VAULT = getVaultAddress(112);
+            IFluidVaultT1(wstUSR_GHO_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(wstUSR_GHO_VAULT).updateLiquidationThreshold(LT);
+        }
+    }
+
+    // @notice Action 4: Set wstUSR smart vaults T3 and WSTUSR / USDTB vault launch limits
+    function action4() internal isActionSkippable(4) {
+        {
+            // dust limits for wstUSR/USDTb vault
+            address wstUSR_USDTb_VAULT = getVaultAddress(142);
+
+            // [TYPE 1] WSTUSR/USDTbvault - Dust limits
+            VaultConfig memory VAULT_wstUSR_USDTb = VaultConfig({
+                vault: wstUSR_USDTb_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: wstUSR_ADDRESS,
+                borrowToken: USDTb_ADDRESS,
+                baseWithdrawalLimitInUSD: 8_000_000, // $8M
+                baseBorrowLimitInUSD: 6_000_000, // $6M
+                maxBorrowLimitInUSD: 20_000_000 // $20M
+            });
+
+            setVaultLimits(VAULT_wstUSR_USDTb);
+            VAULT_FACTORY.setVaultAuth(wstUSR_USDTb_VAULT, TEAM_MULTISIG, false);
+        }
+
+        {
+            // launch limits for wstUSR/USDC-USDT vault
+            address wstUSR_USDC_USDT_VAULT = getVaultAddress(143);
+            address USDC_USDT_DEX = getDexAddress(2);
+
+            {
+                // [TYPE 3] WSTUSR<>USDC-USDT vault - Dust limits
+                VaultConfig memory VAULT_wstUSR_USDC_USDT = VaultConfig({
+                    vault: wstUSR_USDC_USDT_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_3,
+                    supplyToken: wstUSR_ADDRESS, // Set at vault level
+                    borrowToken: address(0), // Set at DEX level
+                    baseWithdrawalLimitInUSD: 8_000_000, // $8M
+                    baseBorrowLimitInUSD: 0,
+                    maxBorrowLimitInUSD: 0
+                });
+
+                setVaultLimits(VAULT_wstUSR_USDC_USDT);
+                VAULT_FACTORY.setVaultAuth(
+                    wstUSR_USDC_USDT_VAULT,
+                    TEAM_MULTISIG,
+                    true
+                );
+            }
+
+            {
+                DexBorrowProtocolConfigInShares
+                    memory config_ = DexBorrowProtocolConfigInShares({
+                        dex: USDC_USDT_DEX,
+                        protocol: wstUSR_USDC_USDT_VAULT,
+                        expandPercent: 30 * 1e2, // 20%
+                        expandDuration: 6 hours, // 6 hours
+                        baseBorrowLimit: 3_000_000 * 1e18, // $6M
+                        maxBorrowLimit: 10_000_000 * 1e18 // $20M
```

Comment:
can adjust these slightly as the yield on this dex already brought exchange price to 2.11
```suggestion
                        baseBorrowLimit: 2_900_000 * 1e18, // $6M
                        maxBorrowLimit: 9_800_000 * 1e18 // $20M
```